client/src/App.tsx

```diff
@@ -3,13 +3,18 @@ import logo from './logo.svg';
 import './App.css';
 
 function App() {
+	function roomAlert() {
+		alert("In the future this button will create a new room. \nBut not today.")
```

Comment:
We use single quotes for strings. Auto-formatting with Prettier should also take care of this.

Reply:
I acknowledge your observation and have now rectified my mistake.